tests/third_party/test_numpy.py

```diff
@@ -0,0 +1,42 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/python-holidays
+#  License: MIT (see LICENSE file)
+
+from platform import platform, python_implementation, python_version
+from unittest import TestCase
+
+import pytest
+
+from holidays.countries.cambodia import Cambodia
+from holidays.countries.thailand import Thailand
+from holidays.countries.ukraine import Ukraine
+
+
+class TestNumpy(TestCase):
+    @pytest.mark.skipif(
+        platform().startswith("Windows")
+        and python_implementation().startswith("PyPy")
+        and python_version().startswith("3.10"),
+        reason='Avoid "make: *** [Makefile:63: test] Error 5" for pypy3.10 on Windows',
+    )
```

Comment:
Without this skip, test also [ran without errors](https://github.com/KJhellico/python-holidays/actions/runs/8928197300/job/24523239200) for me.

Reply:
It's not there yet :)

https://github.com/KJhellico/python-holidays/actions/runs/8928197300/job/24523240416